openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1587,7 +1540,8 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setCertRequestedCallback)(TCN_STDARGS, jlon
         SSL_CTX_set_client_cert_cb(c->ctx, NULL);
     } else {
         jclass callback_class = (*e)->GetObjectClass(e, callback);
-        jmethodID method = (*e)->GetMethodID(e, callback_class, "requested", keyMaterialRequestedMethodSignature);
+        jmethodID method = (*e)->GetMethodID(e, callback_class, "requested","(JJJ[B[[B)V");
```

Comment:
Do you need to check if there was an exception after the GetObjectClass call?

Reply:
I dont think so (also note this did not change):
https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/functions.html

At worse we can do a followup.